src/error.ts

```diff
@@ -688,8 +725,8 @@ export class MongoWriteConcernError extends MongoServerError {
   }
 }
 
-// see: https://github.com/mongodb/specifications/blob/master/source/retryable-writes/retryable-writes.rst#terms
-const RETRYABLE_ERROR_CODES = new Set<number>([
+// https://github.com/mongodb/specifications/blob/master/source/retryable-reads/retryable-reads.rst#retryable-error
+const RETRYABLE_READS_ERROR_CODES = new Set<number>([
```

Comment:
nit: can we use consistent singular/plural form for both reads and writes? i.e., rename this one to `read` or the one below to `writes`

Reply:
done!